interface-definitions/nat66.xml.in

```diff
@@ -151,68 +151,17 @@
                   <help>IPv6 destination prefix options</help>
                 </properties>
                 <children>
-                  <leafNode name="address">
-                    <properties>
-                      <help>IPv6 address or prefix to be translated</help>
-                      <valueHelp>
-                        <format>ipv6</format>
-                        <description>IPv6 address</description>
-                      </valueHelp>
-                      <valueHelp>
-                        <format>ipv6net</format>
-                        <description>IPv6 prefix</description>
-                      </valueHelp>
-                      <valueHelp>
-                        <format>!ipv6</format>
-                        <description>Match everything except the specified IPv6 address</description>
-                      </valueHelp>
-                      <valueHelp>
-                        <format>!ipv6net</format>
-                        <description>Match everything except the specified IPv6 prefix</description>
-                      </valueHelp>
-                      <constraint>
-                        <validator name="ipv6-address"/>
-                        <validator name="ipv6-prefix"/>
-                        <validator name="ipv6-address-exclude"/>
-                        <validator name="ipv6-prefix-exclude"/>
-                      </constraint>
-                    </properties>
-                  </leafNode>
+                  #include <include/nat-address-ipv6.xml.i>
```

Comment:
I have nothing against this refactoring but I have reservations about including it in this PR. Unless I'm missing anything, this change just moves IPv6 address match commands to a reusable template and isn't related or required for the group support logic.

Besides, if we are to move it to a reusable template, we may want to (and likely should) use that template in other places, and that will require a separate PR anyway. Could you keep the changes in this PR to the minimum required to support groups and make a separate PR about the reusable template?

Reply:
I tried to align it a bit with the template of nat44 and make it easier to read. There is currently no other place which would use this template.

I can revert that, no problem.